client/src/pages/Login.tsx

```diff
@@ -26,7 +26,7 @@ const Login = observer(() => {
 
   return (
     <div className="page-container">
-      <div className="page-wraper">
+      <div className="page-wrapper">
```

Comment:
Not sure if this line caused it but can you center the login layout?
![image](https://github.com/assafnoahkoren/quiz-app/assets/16115480/8bedf0ac-fa62-4044-8b59-2032f72dcc62)